python/pyspark/sql/tests/pandas/test_pandas_transform_with_state.py

```diff
@@ -852,13 +872,6 @@ def check_results(batch_df, batch_id):
             StatefulProcessorWithInitialStateTimers(), check_results, "processingTime"
         )
 
-    # run the same test suites again but with single shuffle partition
```

Comment:
Isn't this something we intentionally added because of some observed failure? Mind explaining why we are going to remove this?

Reply:
I am moving this to the end of the test suites for better organization of the file.